<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="forwardLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>przód</string>
    </property>
   </widget>
   <widget class="QLabel" name="backwardLabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>80</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>tył</string>
    </property>
   </widget>
   <widget class="QLabel" name="leftLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>lewo</string>
    </property>
   </widget>
   <widget class="QLabel" name="rightLabel">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>prawo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pictureButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>zdjęcie</string>
    </property>
   </widget>
   <widget class="QLabel" name="pictureView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>420</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>połącz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLine">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>379</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>51716</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipLine">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>339</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>169.254.164.187</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>340</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>port</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraLeftLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kamera lewo</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cameraRightLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>46</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kamera prawo</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
